npm/private/test/snapshots/npm_defs.bzl

```diff
@@ -3088,7 +3088,7 @@ def npm_link_targets(name = "node_modules", package = None, prod = True, dev = T
                 pass
         elif bazel_package == "examples/npm_deps":
             if prod:
-                link_targets.append(":{}/ms".format(name))
```

Comment:
@dzbarsky this one is interesting... changing which dictionaries we iterate over caused this to (correctly, I believe) start being marked as a dev dep...